src/index.js

```diff
@@ -102,6 +104,12 @@ async function writePackageJSON() {
   await writeJsonFile(path, pkg, { detectIndent: true });
 }
 
+async function installDeps() {
```

Comment:
Shall we declare it as `function installDeps` without `async` since there is no async execution in this function? As in we are invoking `crossSpawn.sync`.